docs/api/commands/events.md

```diff
@@ -6,4 +6,4 @@ This command is currently unimplemented. Follow [DOCKER-78](http://smartos.org/b
 
 ## Related
 
-- Insert a list of related Docker and CloudAPI methods here
\ No newline at end of file
+- [`MachineAudit`](https://apidocs.joyent.com/cloudapi/#MachineAudit) and `GET /my/machines/:id/audit` in CloudAPI
```

Comment:
Should we consistently use the CLI names for CloudAPI since that's what we're doing for the Docker commands?

This could be `sdc-getmachineaudit`, I think.